client/src/Validation/validation.js

```
@@ -483,8 +483,71 @@ const notificationValidation = joi.object({
 					"string.uri": "Please enter a valid Webhook URL",
 				}),
 			},
+			{
+				is: "ntfy",
+				then: joi.string().uri().required().messages({
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)